deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/ConvolutionLayer.java

```diff
@@ -162,11 +162,11 @@ public double getLearningRateByParam(String paramName) {
 
     @AllArgsConstructor
     public static class Builder extends FeedForwardLayer.Builder<Builder> {
-        private ConvolutionMode convolutionMode = null;
-        private int[] kernelSize = new int[] {5,5};
-        private int[] stride = new int[] {1,1};
-        private int[] padding = new int[] {0, 0};
-        private AlgoMode cudnnAlgoMode = AlgoMode.PREFER_FASTEST;
+        protected ConvolutionMode convolutionMode = null;
+        protected int[] kernelSize = new int[] {5,5};
+        protected int[] stride = new int[] {1,1};
```

Comment:
Again, this kernel size feels arbitrary. I feel like this should just be soething people are required to define.